authoringAssistant/testCases/methane-emissions-increase.json

```diff
@@ -0,0 +1,19 @@
+{
+  "variables": {
+  },
+  "datasets": [
+    {
+      "var": "tableData",
+      "file": "datasets/methane-emissions"
+    }
+  ],
+  "imports": ["methane"],
+  "expected": "en2020.emissions - en2015.emissions",
+  "paragraph": [
+    {
+      "type": "literal",
+      "value": "Between 2015 and 2020, methane emissions increased across most sectors, with the largest absolute rise seen in the Energy Sector ([REPLACE value=\"9.056956560000003\"] increase)."
```

Comment:
This would be a good candidate for a new kind of example, where we replace “Energy” by the name of the sector with the largest absolute rise. Would you please create an issue for this too?

Reply:
#88

Reply:
It's also related to #76